src/core/searchUtils.js

```diff
@@ -0,0 +1,88 @@
+import { search } from 'core/api';
+import { searchStart, searchLoad, searchFail } from 'core/actions/search';
+
+
+export const paramsToFilter = {
+  app: 'clientApp',
+  category: 'category',
+  q: 'query',
+  type: 'addonType',
+};
+
+export function makeFilters(params) {
```

Comment:
Yeah, they convert query strings (which we try to match to AMO's existing ones as best we can) to the internal identifiers we use (to have nicer variable names) and then back again to the params the AMO API expects (eg. we use `clientApp` internally but the API expects `app`.

I'll add some comments explaining this but also will see about a more descriptive name.